docs/blog/2024-10-03-nx-20-release.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Announcing Nx 20
+slug: announcing-nx-20
+authors: [Mike Hartington]
+tags: [nx, release]
+cover_image: /blog/images/2024-10-03/nx-20-header.png
+---
+
+I know it's hard to believe but Nx 20 is here! There's a lot to cover in this release, so let's just dive right in.
+
+## `@nx` All The Things!
+
+If you've been around the Nx ecosystem for any amount of time, you're probably aware that our plugins were published originally under the `@nrwl` npm scope. Since version 16, we've provided backwards compatibility by reexporting `@nx` scoped packages from `@nrwl`. With Nx 20 though, we'll be officially dropping support for the `@nrwl` scoped packages and they will no longer be published to npm. For projects that still have the older packages in them, our [migration docs][rescope] can guide you to what the new packages are in order for you to update any impacted dependencies.
```

Comment:
```suggestion
If you've been around the Nx ecosystem for any amount of time, you're probably aware that our plugins were published originally under the `@nrwl` npm scope. Since version 16, we've provided backwards compatibility by reexporting `@nx` scoped packages from `@nrwl`. With Nx 20 though, we'll be officially dropping support for the `@nrwl` scoped packages and they will no longer be published to npm. For projects that still have the older packages in them, our [migration docs](/deprecated/rescope) can guide you to what the new packages are in order for you to update any impacted dependencies.
```